guides/common/modules/proc_using-a-synced-kickstart-repository.adoc

```diff
@@ -18,11 +18,11 @@ For {RHEL} 8, ensure that you add both *{RHEL} 8 for x86_64 - AppStream Kickstar
 ifeval::["{build}" == "satellite"]
 +
 If you use a disconnected environment, you must import the Kickstart repositories from a {RHEL} binary DVD.
-For more information, see {BaseURL}content_management_guide/importing-kickstart-repositories_content-management[Importing Kickstart Repositories] in the _Content Management Guide_.
+For more information, see {ContentManagementDocURL}importing-kickstart-repositories_content-management[Importing Kickstart Repositories] in the _Content Management Guide_.
 endif::[]
 +
 . Publish a new version of the Content View where the kickstart repository is added and promote it to a required lifecycle environment.
-For more information, see {BaseURL}content_management_guide/managing_content_views[Managing Content Views] in the _Content Management Guide_.
+For more information, see {ContentManagementDocURL}managing_content_views[Managing Content Views] in the _Content Management Guide_.
```

Comment:
This link for Foreman does not work: 

https://docs.theforeman.org/master/Content_Management_Guide/index-foreman.html#managing_content_views

Reply:
Link for Satellite does not work either: 

https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8/html-single/content_management_guide/index#managing_content_views

Reply:
Fixed,